### ISSUE TITLE
Disable Google Fonts Optimization module when Cloudflare APO is enabled

### ISSUE THREAD

**Issue by pedddro** [wp-media/wp-rocket#5702]
**Is your feature request related to a problem? Please describe.**
Cloudflare APO currently strips Google fonts from Google and hosts them locally.  The preconnect links are unnecessary.

**Describe the solution you'd like**
Disable Gfonts optimization when APO is in use to remove the unnecessary preconnect links.

**Describe alternatives you've considered**

**Additional context**
 There are already warning messages and detection code for APO in Wp Rocket, I guess it would be easy to add this feature.

**Comment by pedddro** [comment]
Cloudflare will add a new [Google fonts optimization free product](https://blog.cloudflare.com/cloudflare-fonts-enhancing-website-privacy-speed/), probably also good to recognize when a user is using it and disable the preconnects.

**Comment by thekendog** [comment]
> Cloudflare will add a new [Google fonts optimization free product](https://blog.cloudflare.com/cloudflare-fonts-enhancing-website-privacy-speed/), probably also good to recognize when a user is using it and disable the preconnects.

Yeah I've started using Cloudflare fonts and a simple setting to disable the Google fonts optimization is badly needed.  Right now I have to remove the resource hint using a filter and then also disable the option using WP-CLI.  I deal with A LOT of sites and a checkbox toggle to turn it off would make things much easier.
